src/tools/list-limbo-balance-accounts.ts

```diff
@@ -0,0 +1,67 @@
+// This script is expected to run against a parachain network (using launch.ts script)
+import yargs from "yargs";
+import { table } from "table";
+
+import { getApiFor, NETWORK_YARGS_OPTIONS } from "..";
+import { bnMax } from "@polkadot/util";
+import Web3 from "web3";
+
+const argv = yargs(process.argv.slice(2))
+  .usage("Usage: $0")
+  .version("1.0.0")
+  .options({
+    ...NETWORK_YARGS_OPTIONS,
+  }).argv;
+
+const main = async () => {
+  // Instantiate Api
+  const api = await getApiFor(argv);
+
+  const [allLocks] = await Promise.all([api.query.balances.locks.entries()]);
```

Comment:
Is this working on Moonbeam ?

Reply:
Yes, in a couple of minutes should be done. It shows that Bware Labs collator is already affected https://moonbeam.subscan.io/account/0xC26694a78039548b9AeE363b1e12dEb0F98f44Dc